modules/lx-autoscale/watchmaker-lx-autoscale.template.cfn.yaml

```diff
@@ -1013,13 +1021,12 @@ Resources:
               #!/bin/bash -xe
 
               # Format and mount app volume
-              APPVOL_DEVICE="$(ls /dev/xvdf /dev/nvme1n1 2> /dev/null || true)"
-              if [[ -n "$APPVOL_DEVICE" && "${AppVolumeDevice}" == "true" ]]
+              if [[ "${AppVolumeDevice}" == "true" ]]
```

Comment:
I don't think this works for Nitro-instance types, e.g. t3.*? These types all have nvme disks...

```
ls /dev/xvdf /dev/nvme1n1
```

That should return either `/dev/xvdf` if it exists, OR `/dev/nvme1n1` if it exists. That way it works for older instances types as well as newer ones.

Reply:
What's happening currently is it returns both which causes the error.  `APPVOL_DEVICE=/dev/xvdf /dev/nvme1n1` is the result.

What I'm observing is that a symlink is created using the device name provided at launch so that it automatically links to what the underlying device type is given the chosen instance type.

Reply:
Aha ok. That's nifty. I bet an update to cloud-init is injecting that.